scripts/rollup/build.js

```diff
@@ -365,139 +332,154 @@ async function createBundle(bundle, bundleType) {
     module => !importSideEffects[module]
   );
 
+  const rollupConfig = {
+    input: resolvedEntry,
+    pureExternalModules,
+    external(id) {
+      const containsThisModule = pkg => id === pkg || id.startsWith(pkg + '/');
+      const isProvidedByDependency = externals.some(containsThisModule);
+      if (!shouldBundleDependencies && isProvidedByDependency) {
+        return true;
+      }
+      return !!peerGlobals[id];
+    },
+    onwarn: handleRollupWarning,
+    plugins: getPlugins(
+      bundle.entry,
+      externals,
+      bundle.babel,
+      filename,
+      bundleType,
+      bundle.global,
+      bundle.moduleType,
+      bundle.modulesToStub
+    ),
+    // We can't use getters in www.
+    legacy: bundleType === FB_DEV || bundleType === FB_PROD,
+  };
+  const [mainOutputPath, ...otherOutputPaths] = Packaging.getBundleOutputPaths(
+    bundleType,
+    filename,
+    packageName
+  );
+  const rollupOutputOptions = getRollupOutputOptions(
+    mainOutputPath,
+    format,
+    peerGlobals,
+    bundle.global
+  );
+
   console.log(`${chalk.bgYellow.black(' BUILDING ')} ${logKey}`);
   try {
-    const result = await rollup({
-      input: resolvedEntry,
-      pureExternalModules,
-      external(id) {
-        const containsThisModule = pkg =>
-          id === pkg || id.startsWith(pkg + '/');
-        const isProvidedByDependency = externals.some(containsThisModule);
-        if (!shouldBundleDependencies && isProvidedByDependency) {
-          return true;
-        }
-        return !!peerGlobals[id];
-      },
-      onwarn: handleRollupWarnings,
-      plugins: getPlugins(
-        bundle.entry,
-        externals,
-        bundle.babel,
-        filename,
-        bundleType,
-        bundle.global,
-        bundle.moduleType,
-        bundle.modulesToStub
-      ),
-      // We can't use getters in www.
-      legacy: bundleType === FB_DEV || bundleType === FB_PROD,
-    });
-    await result.write(
-      getRollupOutputOptions(
-        filename,
-        format,
-        bundleType,
-        peerGlobals,
-        bundle.global,
-        bundle.moduleType
-      )
-    );
-    await Packaging.createNodePackage(
-      bundleType,
-      packageName,
-      filename,
-      npmPackagesTmpDir
-    );
-    console.log(`${chalk.bgGreen.black(' COMPLETE ')} ${logKey}\n`);
+    const result = await rollup(rollupConfig);
+    await result.write(rollupOutputOptions);
   } catch (error) {
-    if (error.code) {
-      console.error(
-        `\x1b[31m-- ${error.code}${error.plugin ? ` (${error.plugin})` : ''} --`
+    console.log(`${chalk.bgRed.black(' OH NOES! ')} ${logKey}\n`);
```

Comment:
nit: `console.error` ?

Reply:
There's a further console.error too so I guess it doesn't matter? The diff makes it look like I replaced one with the other, but I just added an additional one.

Reply:
To be honest I'm not even sure what the difference on Node is 😅

Reply:
I think the only difference is that one outputs to `stderr` and the other to `stdout`. I don't think it actually matters in this case. 😁